packages/react-scripts/package.json

```diff
@@ -32,8 +32,8 @@
     "@babel/eslint-parser": "^7.13.14",
     "@pmmmwh/react-refresh-webpack-plugin": "0.4.3",
     "@svgr/webpack": "5.5.0",
-    "@typescript-eslint/eslint-plugin": "^4.5.0",
-    "@typescript-eslint/parser": "^4.5.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
```

Comment:
do you need to bump the other deps besides `eslint-plugin-testing-library` ?

Reply:
This way they're all on latest by default